samples/Paging/Specifications/ContinuationTokenSpecification.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.CosmosRepository.Specification;
+
+namespace Paging.Specifications
+{
+    internal class ContinuationTokenSpecificationImplementation : ContinuationTokenSpecification<Person>
+    {
+        public ContinuationTokenSpecificationImplementation(string continutationToken, int pageSize):
```

Comment:
```suggestion
        public ContinuationTokenSpecificationImplementation(string continuationToken, int pageSize):
```

Reply:
In fact, can we delete this now we have the implementation of this inside the library?